### PR TITLE
Added qt prefix path for qtkeychain and desktop build

### PR DESCRIPTION
**Pull request by Janl1**
Adding QT_PREFIX and QTKEYCHAIN_PREFIX as annotated in https://github.com/nextcloud/desktop/pull/2603 fixes https://github.com/nextcloud/client-building/issues/48

### CODE REVIEW

File: single-build-desktop.bat [hunk 1]
```diff
@@ -71,7 +71,6 @@ call :testEnv QT_PATH
 call :testEnv QT_BIN_PATH
 call :testEnv VCINSTALLDIR
 call :testEnv Png2Ico_EXECUTABLE
-call :testEnv QTKEYCHAIN_CMAKE_DIR
```
Comment (er-vin) [inline single-build-desktop.bat:74]: Shouldn't that variable be cleaned up in build-desktop.bat then?

Comment (Janl1) [reply single-build-desktop.bat:74]: Yes you are right.. I missed that

**Comment by er-vin** [comment]
Thanks, @Janl1. Could you please squash it all in a single commit?

**Comment by Janl1** [comment]
I am not quite sure what to do, sorry - Can't you squash the commits while merging? With Gitlab this is a thing

**Comment by er-vin** [comment]
No worries, I'll deal with it on my end, hopefully I can push to your branch. :-)

**Review by er-vin — APPROVED** [review]

**Pull request merged**